cupyx/scipy/sparse/csr.py

```diff
@@ -6,6 +6,7 @@
 
 import cupy
 from cupy import cusparse
+from cupy.cuda.cub import device_csrmv
```

Comment:
Bug fix?
```suggestion
if cupy.cuda.cub_enabled:
    from cupy.cuda.cub import device_csrmv
```

Reply:
Thanks. I wonder if it might be worth having a separate `cub_enabled` flag for sparse matrix vs. reductions? I guess we can always implement that later, if needed, once we have another alternative such as the potential use of the new cuSPARSE API.